cmd/documentation/documentation.go

```diff
@@ -77,15 +108,33 @@ type Attributes struct {
 }
 
 func main() {
-	verbose := flag.Bool("v", false, "Verbose output")
-
+	flag.BoolVar(&verbose, "v", false, "Verbose output")
+	flag.StringVar(&toolDir, "tool", "", "pass in the documentation tool directory if outside tool folder")
```

Comment:
Please change "tool" to "tooldir", apart from that have tested it and works well!